src/config/build_config.py

```diff
@@ -1,7 +1,7 @@
 # Frameworks for which you want to disable both builds and tests
```

Comment:
Can't add comments to binary file, so adding it here -- Is it necessary to upload `sagemaker_tests.tar.gz` to GitHub?

Reply:
accidental addition. removing it